src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -469,9 +469,10 @@ DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId, char *sizeQ
 
 	List *shardIntervalsOnNode = ShardIntervalsOnWorkerGroup(workerNode, relationId);
 
+	bool skipPartitions = false;
```

Comment:
wouldn't skipping partitions make sense here as well? I bet we should have a similar problem here as well?